lib/rubocop/target_ruby.rb

```diff
@@ -7,7 +7,7 @@ class TargetRuby
     DEFAULT_VERSION = KNOWN_RUBIES.first
 
     OBSOLETE_RUBIES = {
-      1.9 => '0.50', 2.0 => '0.50', 2.1 => '0.58', 2.2 => '0.69', 2.3 => '0.82'
+      1.9 => '0.50', 2.0 => '0.50', 2.1 => '0.58', 2.2 => '0.69', 2.3 => '0.81'
```

Comment:
It has been specified the RuboCop version with a Ruby version support dropped. Could you tell me why the version is wrong?

On the other hand, I noticed that the specifications are not consistent with Ruby 1.9 and Ruby 2.0. Below are the respective Ruby versions and the RuboCop versions that dropped it.

- Ruby 2.3 (0.82) https://github.com/rubocop-hq/rubocop/commit/5873894
- Ruby 2.2 (0.69) https://github.com/rubocop-hq/rubocop/commit/4910d51
- Ruby 2.1 (0.58) https://github.com/rubocop-hq/rubocop/commit/a707c23
- Ruby 2.0 (0.51) https://github.com/rubocop-hq/rubocop/commit/7532279
- Ruby 1.9 (0.42) https://github.com/rubocop-hq/rubocop/commit/9d27451

Reply:
> It has been specified the RuboCop version with a Ruby version support dropped

The name `OBSOLETE_RUBIES` is not very clear, but it indicates the last version where a Ruby version is supported.

The message says: "2.3-compatible analysis was dropped after version #{last_version}.".

The method using it is called `rubocop_version_with_support`.

Reply:
Note that this is the information that is actually helpful, i.e. "hey, I want to maintain my ruby_target so what version of RuboCop can I upgrade to then?"

Reply:
Final note: I didn't check the other Ruby versions than 2.3